modules/ROOT/pages/rtfctl-permissions.adoc

```diff
@@ -0,0 +1,112 @@
+= RTFCTL Kubernetes Permissions
+
+Understanding and enabling the specific permissions required over the Kubernetes resources is crucial for successfully running `rtfctl` commands. Properly configured role permissions are essential for seamless execution of `rtfctl` operations.
```

Comment:
I'm not sure what the first sentence means: "permissions required over the Kubernetes resources".
Do you need to introduce rtfctl? What is it and why do you need permissions?

If you flip these around and make them active - what does the user need to know and do?

To do xxx (run rtfctl to configure Kubernetes resources?), you must understand and enable specific permissions.

The second sentence is redundant with the first.

Reply:
Global - if the UI uses this camelCase, then keep. Otherwise, use English for these. For example "API groups"